Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableEntitiesManager.h

```diff
@@ -224,6 +234,8 @@ namespace AzFramework
         //! SpawnablePriority_Default which gives users a bit of room to fine tune the priorities as this value can be configured
         //! through the Settings Registry under the key "/O3DE/AzFramework/Spawnables/HighPriorityThreshold".
         SpawnablePriority m_highPriorityThreshold { 64 };
+    private:
+        SpawnedEntityTicketMapper m_spawnedEntityTicketMapper;
```

Comment:
This shouldn't be on the Spawnable Entity Manager, but rather on the Spawnable System Component as that's where interfaces are set up.

Reply:
Since this mapper only deals with entities spawned through SEM, it made sense to construct it here. Also, if this is defined here, we can directly use the object instead of trying to fetch the interface every time we need to use the map

Reply:
I can see the optimization options, although caching the pointer to the interface is alternative option and getting an interface pointer isn't too expensive either. The issue I see is that it spreads out where interfaces are assigned in multiple places. Keeping things clean by assigning interfaces in a system component makes it easier to find where and when an interface is initialized.

Reply:
Not needed anymore.